### ISSUE TITLE
continuous generation / ambient mode ?

### ISSUE THREAD

**Issue by beppe9000** [pernyblom/pernyblom.github.io#5]
is there a way to increase song length to infinity and play it as it is generated ??

**Comment by pernyblom** [comment]
No, but it should be possible, with minor coding, to generate another new song in advance while another song plays and just change some seed to make them sound similar.
